README.md

```diff
@@ -147,56 +197,46 @@ models:
     quickbooks_source:
       +schema: my_new_schema_name # leave blank for just the target_schema
 ```
-### Disabling models
-
-This package takes into consideration that not every QuickBooks account utilizes the same transactional tables, and allows you to disable the corresponding functionality. By default, most variables' values are assumed to be `true` (with exception of purchase orders and credit card payment transactions). Add variables for only the tables you want to disable or enable respectively:
 
+### Change the source table references
+If an individual source table has a different name than the package expects, add the table name as it appears in your destination to the respective variable:
+> IMPORTANT: See this project's [`dbt_project.yml`](https://github.com/fivetran/dbt_quickbooks/blob/main/dbt_project.yml) variable declarations to see the expected names.
+    
 ```yml
-# dbt_project.yml
-
-...
 vars:
-  using_address:        false         #disable if you don't have addresses in QuickBooks
-  using_bill:           false         #disable if you don't have bills or bill payments in Quickbooks
-  using_credit_memo:    false         #disable if you don't have credit memos in Quickbooks
-  using_department:     false         #disable if you don't have departments in Quickbooks
-  using_deposit:        false         #disable if you don't have deposits in Quickbooks
-  using_estimate:       false         #disable if you don't have estimates in Quickbooks
-  using_invoice:        false         #disable if you don't have invoices in Quickbooks
-  using_invoice_bundle: false         #disable if you don't have invoice bundles in Quickbooks
-  using_journal_entry:  false         #disable if you don't have journal entries in Quickbooks
-  using_payment:        false         #disable if you don't have payments in Quickbooks
-  using_refund_receipt: false         #disable if you don't have refund receipts in Quickbooks
-  using_transfer:       false         #disable if you don't have transfers in Quickbooks
-  using_vendor_credit:  false         #disable if you don't have vendor credits in Quickbooks
-  using_sales_receipt:  false         #disable if you don't have sales receipts in QuickBooks
-  using_purchase_order: true          #enable if you want to include purchase orders in your staging models
-  using_credit_card_payment_txn: true #enable if you want to include credit card payment transactions in your staging models
+    quickbooks_<default_source_table_name>_identifier: your_table_name 
 ```
 
-## Analysis
+## Analysis of the financial models
```

Comment:
I think this would actually be a good separate step. Maybe something like `(Optional) Step 6: Validate your data`

Reply:
Updated!